index.bs

```diff
@@ -9560,8 +9560,8 @@ within a special scope named {{AudioWorkletGlobalScope}}.
 
 Each {{BaseAudioContext}} possesses exactly one
 {{AudioWorklet}}. Scripts imported using this {{AudioWorklet}}
-are run in one or more {{AudioWorkletGlobalScope}} global scopes,
-which are created to process audio associated with that
+are run in one {{AudioWorkletGlobalScope}} global scope,
```

Comment:
I think we can remove "global scope" at the end.

Reply:
Done.